app/graphql/loaders/fancy_loader/type_generator.rb

```diff
@@ -0,0 +1,41 @@
+##
+# Generate parameter types for sorting
+class Loaders::FancyLoader::TypeGenerator
+  class SortDirection < GraphQL::Schema::Enum
+    graphql_name 'SortDirectionEnum'
+
+    value :ASCENDING, value: :asc
+    value :DESCENDING, value: :desc
+  end
```

Comment:
can we move to a separate file, this is gross :sob:

Reply:
Okay that's fair lol idk why I crammed it in here